updated Readme (#3)
pokt-network/rain-tree-sim: python/README.md

### Install Dependencies

Make sure you install the corresponding dependencies.

```bash
    pip3 install -r python/requirements.txt
```

### Using the Python Simulator 

#### Prerequisite:

> - Python: `v 3.10.8` 

Reviewer: or greater?

- pip3

### Setup local Environment:
To run the simulator, do the following:
1. Download repo: `git clone https://github.com/pokt-network/rain-tree-sim.git`
2. `cd python` 

> 3. Install the corresponding dependencies `pip3 install -r requirements.txt`

Reviewer: What to you think of just simplifying it to this;

---

To run the simulator, do the following:
```bash
git clone https://github.com/pokt-network/rain-tree-sim.git
cd python
pip3 install -r requirements.txt
```

### Run Simulator
Run the following command:

> `rainTreeTestOutputFilename=/tmp/answer.go numRainTreeNodes=12 make p2p_test_generator`

Reviewer: This makes it easier to copy-pasta

---

Run the following command:
```bash
rainTreeTestOutputFilename=/tmp/answer.go numRainTreeNodes=12 make p2p_test_generator
```

> If you wish to run different scenarios, You can modify two parameters to the `p2p_test_generator` make target:

Reviewer: ```suggestion
If you wish to run different scenarios, you can modify two parameters to the `p2p_test_generator` make target:
```

- `rainTreeTestOutputFilename`: the file where the unit test should be written to
- `numRainTreeNodes`: the number of nodes to run in the RainTree simulation

> ### See the results:

Reviewer: NIT: Titles shouldn't have `:`.

Leaving one comment but ditto elsewhere

By default, the test stores the results at: `/tmp/answer.go`. 

> Through termainal, you can open it with: `code /tmp/answer.go` 

Reviewer: ```suggestion
Through termainal, you can open view it with: `cat /tmp/answer.go` 
```

If you're using VS-Code, you can open it using: `code /tmp/answer.go` 

**Note: if you are going to be running the Pocket Unit test with the results, leave this file open and read the section below.**  

> ### Running the Pocket Unit Test:

Reviewer: ```suggestion

### Running the Pocket Unit Test:
```

> You can run the p2p unit test in the [Pocket repo](https://github.com/pokt-network/pocket/blob/main/p2p/module_raintree_test.go), which will break down the messages that is being sent from one node to another from the pre-defined raintree test as well as the unique test generated from the python simulator.  

Reviewer: ```suggestion
You can run the p2p unit test in the [Pocket repo](https://github.com/pokt-network/pocket/blob/main/p2p/module_raintree_test.go), which will break down the messages that is being sent from one node to another from the pre-defined RainTree test as well as the unique test generated from the Python simulator.  
```

#### Prerequisite:
Before you can run the unit test, you will first need to clone the [Pocket repo](https://github.com/pokt-network/pocket) and follow the local env setup steps outlined [here](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md#lfg---development). 

#### Running Unit Test: 

1. Open up the output file located: `/tmp/answer.go` 

> 2. Add comments to the tree visualization(see below)

Reviewer: 🔥
